dart/lib/src/protocol/sentry_event.dart

```diff
@@ -30,17 +30,17 @@ class SentryEvent {
     this.level,
     this.culprit,
     this.user,
-    Contexts contexts,
+    Contexts? contexts,
     this.request,
     this.debugMeta,
   })  : eventId = eventId ?? SentryId.newId(),
         timestamp = timestamp ?? getUtcDateTime(),
         contexts = contexts ?? Contexts(),
         modules = modules != null ? Map.from(modules) : null,
-        tags = tags != null ? Map.from(tags) : null,
-        extra = extra != null ? Map.from(extra) : null,
-        fingerprint = fingerprint != null ? List.from(fingerprint) : null,
-        breadcrumbs = breadcrumbs != null ? List.from(breadcrumbs) : null;
+        tags = Map.from(tags ?? {}),
+        extra = Map.from(extra ?? {}),
+        fingerprint = List.from(fingerprint ?? []),
+        breadcrumbs = List.from(breadcrumbs ?? []);
```

Comment:
similar to comment above, well I wont comment on the next ones, if we decide doing like this, we could make maps and lists non nullable and only add to json if non empty